harness/determined/keras/callbacks.py

```diff
@@ -172,6 +207,7 @@ def on_train_batch_end(self, batch: int, logs: Optional[Dict] = None) -> None:
 
         self.state.total_batches += 1
         self.state.epoch_batches += 1
+        self.state.validation_batches += 1
```

Comment:
non-blocking: calling it `batches_since_validation_period` may be more clear

Reply:
(no longer relevant)